### PR TITLE
Ux/misc improvments

### PR DESCRIPTION
**Pull request by HenriChabert**

### CODE REVIEW

File: resource/labeling-app/components/category-selector.js [hunk 1]
```diff
@@ -205,7 +205,7 @@ let CategorySelector = {
                     </div>
                     <div v-else style="width: 100%;">
                         <h2 v-if="status !== 'SKIPPED'">No labels yet</h2>
-                        <p v-if="type === 'image-object'" style="margin: auto;">Select a category by ...</p>
+                        <p v-if="isMultiLabel" style="margin: auto;">Select a category by ...</p>
                         <div class="circles-container">
                             <div style="max-width: 110px;">
                                 <div class="circle cat-example"></div>
```
Comment (andreybavt) [inline resource/labeling-app/components/category-selector.js:211]: I think we're still missing "or" between bubbles
image

File: resource/labeling-app/components/utils/utils.js [hunk 2]
```diff
@@ -81,4 +81,17 @@ export function loadConfig() {
     });
 }
 
+export function shortcut(event) {
+    const shortcuts = {
+        'multi-selection': [event.ctrlKey, event.metaKey, event.shiftKey],
+        'delete': [event.key === "Delete", event.key === "Backspace"],
+        'back': [event.code === 'ArrowLeft'],
+        'next': [event.code === 'ArrowRight', event.code === 'Enter'],
+        'skip': [event.code === 'Space']
```
Comment (andreybavt) [inline resource/labeling-app/components/utils/utils.js:90]: This should be `tab` and space for save&next

**Review by andreybavt — CHANGES_REQUESTED** [review]

File: resource/labeling-app/components/category-selector.js (hunk 1 above)
Comment (HenriChabert) [reply resource/labeling-app/components/category-selector.js:211]: Fixed in af87c6a

File: resource/labeling-app/components/utils/utils.js (hunk 2 above)
Comment (HenriChabert) [reply resource/labeling-app/components/utils/utils.js]: Fixed in f354aad

**Review by andreybavt — APPROVED** [review]

**Pull request merged**